packages/ui-components/src/components/ComboBox/ComboBox.test.js

```diff
@@ -533,6 +533,12 @@ describe("ComboBox", () => {
     expect(cbox).toHaveValue("option 2")
   })
 
+  test("renders a wrapperClassName to the outer wrapping <div> element", async () => {
```

Comment:
Do you need to make this test async? cause the lint going to fail with error async no await(when we move this class to ts)